lib/services/identity/controllers/identity-config-controller.js

```diff
@@ -17,6 +19,24 @@ class IdentityConfigController {
 		this.bus
 			.query({role: 'identity', cmd: 'config'}, {channel, platform, user})
 			.then(resource => {
+				if (user) {
+					return resource;
+				} else { // eslint-disable-line no-else-return
+					const id = uuid.v4();
+					const user = {id, channel: channel.id, type: 'user'};
+					const payload = {channel: channel.id, platform: platform.id, user: id, audience: ['platform']};
+
+					return Promise.join(
+						this.bus.sendCommand({role: 'store', cmd: 'set', type: 'user'}, user),
+						this.bus.query({role: 'identity', cmd: 'sign'}, payload),
+						(user, jwt) => {
+							resource.jwt = jwt;
```

Comment:
Maybe add `resource.user = user` here ? I added to the config a while back, so just to be consistent.